Dockerfile

```
@@ -16,3 +16,4 @@ EXPOSE 8001
 STOPSIGNAL SIGINT
 
 ENTRYPOINT [ "python", "start.py" ]
+HEALTHCHECK --interval=5s --timeout=3s CMD /home/warrior/data/wget-at -nv -t1 'http://localhost:8001/index.html' --delete-after || exit 1
```

```suggestion
HEALTHCHECK --interval=5s --timeout=3s CMD /home/warrior/data/wget-at -nv -t1 'http://localhost:8001/index.html' -O - 1> /dev/null || exit 1
```
This avoids downloading and deleting the file and instead just pipes the contents to `/dev/null`

---

`-O /dev/null` to avoid having to write to stdout and redirect stdout to /dev/null ;)

---

also, mind updating the other dockerfiles as well (even though i think they're not used)?

---

When suggested changes are in by @andrew-stclair, this PR will be merged.

---

Suggestions have been implemented